app/konnect/analytics/generate-reports.md

```diff
@@ -74,7 +73,7 @@ and click it again to show the item. The graph resets on a refresh.
 
 You can export any custom report in CSV format.
 
-To export a report, open {% konnect_icon vitals %} [Vitals](https://cloud.konghq.com/vitals), click **Reports**, then follow these steps:
+To export a report, open {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics), click **Reports**, then follow these steps:
```

Comment:
do we know for sure that the URLs in Konnect are also changing with this first round?

Reply:
yes